src/environments/environment.ts

```diff
@@ -5,6 +5,8 @@
 export const environment = {
   production: false,
   isDevelop: false,
+  isIntegration: false,
+  isPreProd: false,
```

Comment:
this ought to be one variable named environment: which contains the properties 'production', 'develop', 'integration' and 'preprod'

Reply:
Agreed, changed to use environment